analyzer/src/main/kotlin/managers/utils/DependencyGraphBuilder.kt

```diff
@@ -153,9 +155,9 @@ class DependencyGraphBuilder<D>(
     )
 
     /**
-     * Return a set with all the packages that have been encountered for the current project.
+     * Return a sorted set with all the packages that have been encountered for the current project.
      */
-    fun packages(): Set<Package> = resolvedPackages
+    fun packages(): SortedSet<Package> = resolvedPackages.toSortedSet()
```

Comment:
I'm wondering whether we should do somewhat the opposite, and convert to a sorted set as late as possible. The reason is that I recall we had performance issues when inserting into sorted sets. And as we only need the sorting in user-facing output / the serialized ORT result files, I wonder whether we could e.g. instead make `PackageManagerResult` not expect a sorted set, and do the sorting later. Or is that class already part of the serialization?

Reply:
No, this class is not serialized. I like this idea. The creation of the sorted set can then be deferred until the analyzer result is constructed by the corresponding builder.
I would then close this PR and open a new one following this other approach, okay?

Reply:
Yes, fine with me!